src/components/Practice/PhrasesInAction.jsx

```
@@ -47,6 +48,302 @@ import { Log } from "../../services/telemetryService";
+        display: "flex",
+        alignItems: "center",
+        justifyContent: "center",
+        gap: "20px",
+        marginTop: isMobile ? "0" : "30px",
+      }}
+    >
+      {language !== "en" && recordedBlob && (
+        <div
+          role="button"
+          tabIndex={0}
+          onClick={isPlayingRecordedAudio ? stopRecordedAudio : playRecordedAudio}
```

ESLint

[error] Replace `isPlayingRecordedAudio·?·stopRecordedAudio·:·playRecordedAudio` with `⏎············isPlayingRecordedAudio·?·stopRecordedAudio·:·playRecordedAudio⏎··········`

(prettier/prettier)